lib/reporters/mini.js

```diff
@@ -70,19 +70,16 @@ MiniReporter.prototype.prefix = function (str) {
 };
 
 MiniReporter.prototype._test = function (test) {
-	var title;
+	var title = test.title;
 
 	if (test.todo) {
-		title = colors.todo('- ' + test.title);
```

Comment:
> If a human can not see todo and skip tests, shouldn't we also remove color from there?

That's what this PR does.

> But wait, if the next test is slow, user theoretically can actually see skip/todo tests. So I guess we should keep the prefix and color.

How about we just not show them all together? So if a slow test is running, the last actual test is shown, not todo/skip.